doc/specs/stdlib_io.md

```diff
@@ -273,3 +273,45 @@ program demo_getline
     end do
 end program demo_getline
 ```
+
+## Formatting constants
+
+### Status
+
+Experimental
+
+### Description
+
+Formatting constants for printing out integer, floating point, and complex numbers at their full precision.
+Provides formats for all kinds as defined in the `stdlib_kinds` module.
+
+### Example
+
+```fortran
+program demo_fmt_constants
+    use, intrinsic :: iso_fortran_env, only : int32, int64, real32, real64
```

Comment:
As the desctiption mentions "all kinds as defined in the `stdlib_kinds` module, I suggest to use the kinds defined in stdlib_kinds, instead of those defined in `iso_fortran_env`. Otherwise, it may look strange.

Reply:
Good catch! Fixed.